packages/expect/src/__tests__/__snapshots__/matchers.test.js.snap

```diff
@@ -2150,6 +2150,21 @@ exports[`.toEqual() {pass: false} expect([97, 98, 99]).toEqual([97, 98, 100]) 1`
 <d>  ]</>
 `;
 
+exports[`.toEqual() {pass: false} expect([undefined, undefined, 1, undefined]).toEqual([undefined, undefined, 2, undefined]) 1`] = `
+<d>expect(</><r>received</><d>).</>toEqual<d>(</><g>expected</><d>) // deep equality</>
+
+<g>- Expected  - 1</>
+<r>+ Received  + 1</>
```

Comment:
The Expected / Received seems to be wrong there 🤔 

Nonetheless it does not seem related to my change as I got the same snapshot before my dev.

Reply:
wrong how? they reflect the `+` and `-` in the diff below, no?

Reply:
```
<d>  Array [</>
<d>    undefined,</>
<d>    undefined,</>
<g>-   2,</>
<r>+   1,</>
<d>    undefined,</>
<d>  ]</>
```

It looks that the diff should be:

```
<g>- Expected  - 2</>
<r>+ Received  + 1</>
```

(clearly not sure of me on that one)

Reply:
ah, I see what you mean. You misunderstand (I think 😛) - the heading is a counter of the number of `-` and `+` lines, it's not an extracted part of the diff itself